### PR TITLE
Add org.olivevideoeditor.Olive

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build org.olivevideoeditor.Olive
testbot, build org.olivevideoeditor.Olive

File: org.olivevideoeditor.Olive.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=OLIVE_EFFECTS_PATH=/app/share/olive-editor/effects
+cleanup:
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: 'x264'
```
Comment (barthalion) [inline org.olivevideoeditor.Olive.yaml]: I don't think it has to be quoted – but it doesn't really matter.

Comment (eszlari) [reply org.olivevideoeditor.Olive.yaml]: It makes the syntax highlighter of Github happy.

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.olivevideoeditor.Olive

@itsmattkc, you should receive an invitation to collaborators. Please make sure to accept it to get write access to the above repository.

Thanks!

**Comment by itsmattkc** [comment]
I'll give this a proper test tomorrow, but thanks for setting it up!

I guess the Qt and/or OpenGL implementation on arm is different somehow? I'm not familiar with those architectures at all.

**Comment by itsmattkc** [comment]
It's looking good, what are the next steps?

**Comment by barthalion** [comment]
Nothing, it's all done!

https://flathub.org/apps/details/org.olivevideoeditor.Olive

From now on, you can just push your changes to the repository linked above. You may also want to read wiki page on [app maintenance](https://github.com/flathub/flathub/wiki/App-Maintenance). If you are new to flatpak in general, initial setup is also [documented](https://flatpak.org/setup/), also for [flatpak-builder](https://docs.flatpak.org/en/latest/first-build.html).

**Comment by itsmattkc** [comment]
Okay awesome, thanks!

**Pull request closed without merging**